lib/ancestry/has_ancestry.rb

```diff
@@ -44,16 +44,14 @@ def has_ancestry options = {}
     scope :subtree_of, lambda { |object| where(to_node(object).subtree_conditions) }
     scope :siblings_of, lambda { |object| where(to_node(object).sibling_conditions) }
     scope :ordered_by_ancestry, lambda {
-      if %w(mysql mysql2 sqlite postgresql).include?(connection.adapter_name.downcase) &&
-        defined?(ActiveRecord.version) && ActiveRecord.version.to_s >= "5"
+      if %w(mysql mysql2 sqlite postgresql).include?(connection.adapter_name.downcase) && ActiveRecord::VERSION::MAJOR >= 5
```

Comment:
I was thinking of moving this logic to a class method we can memoize and that describes *why* we are doing this.  From reading the next line it seems it's to support the `coalesce` operator?  Is that true, and if so, why the need for the Rails 5 check?